scripts/bisect.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+echo $(pwd)
+
+
+while getopts "e:f:hl:t:" opt; do
+  case "$opt" in 
+    e)
+      exp_file="$OPTARG"
+      echo "exp file is $exp_file"
+      ;;
+    l)
+      libc="$OPTARG"
+      echo "libc is $libc"
+      ;;
+    f)
+      test_case="$OPTARG"
+      echo "test case is $test_case"
+      ;;
+    t)
+      target_arch="$OPTARG"
+      echo "target arch is $target_arch"
+      ;;
+    ?|h)
+      echo "Usage: $(basename $0) -l <libc> -e <exp file> -f <test file> -t <target arch>"
```

Comment:
Add an exit 0 or exit 1 statement, we don't want to continue executing when the user asks for help